spec/dummy_app/app.rb

```diff
@@ -0,0 +1,53 @@
+require 'active_record'
+require_relative 'schema_helper'
+
+class App
+  class << self
+    def init
+      database_connect
+      generate_schema
+      load_models
+      create_models
+    end
+
+    def database_connect
+      SchemaHelper.connect_to('sqlite3', ':memory:')
```

Comment:
I kept sqlite3 here to avoid loosing time on setting up a pg. The whole `fake_database` could be worked around in different way, I pragmatically sticked with [filterable](https://github.com/toschas/filterable/) approach.